### PR TITLE
Ansible/install jenkins properly

### PR DESCRIPTION
**Pull request by Cavonstavant**

### CODE REVIEW

File: ansible/roles/jenkins/tasks/gcloud.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    name: google-cloud-cli
+    state: present
+  become: true
+
+- name: Configure gcloud for jenkins account
+  become: true
+  become_user: jenkins
+  block:
+    - name: Activate service account
+      ansible.builtin.command: gcloud auth activate-service-account --key-file={{ gcp.service_account_key_file }}
```
Comment (majent4) [inline ansible/roles/jenkins/tasks/gcloud.yml:36]: Service account url argument and --project option are missing

Comment (majent4) [reply ansible/roles/jenkins/tasks/gcloud.yml:36]: For reference :
https://github.com/TEAM-AAAAAAAAAAAAAAAA/Whanos/blob/main/scripts/gcloud-setup.sh#L10

Comment (Cavonstavant) [reply ansible/roles/jenkins/tasks/gcloud.yml:36]: My bad

File: ansible/roles/jenkins/tasks/tools.yml [hunk 2]
```diff
@@ -37,7 +37,42 @@
...
+  ansible.builtin.get_url:
+    url: https://dl.k8s.io/release/v1.26.0/bin/linux/amd64/kubectl
+    dest: /tmp/kubectl
+    mode: 0755
+  changed_when: false
+
+- name: Install kubectl
+  ansible.builtin.command:
+    cmd: sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+    creates: /usr/bin/kubectl
```
Comment (majent4) [inline ansible/roles/jenkins/tasks/tools.yml:76]: What's the purpose of the creates field ?

Comment (majent4) [inline ansible/roles/jenkins/tasks/tools.yml:76]:
```suggestion
    creates: /usr/local/bin/kubectl
```

**Pull request merged**